source/usage-examples/distinct.txt

```diff
@@ -0,0 +1,76 @@
+===================================
+Retrieve Distinct Values of a Field
+===================================
+
+.. default-domain:: mongodb
+
+You can retrieve a list of distinct values for a field across a
+collection by calling the ``distinct()`` method on a ``MongoCollection``
+object. Pass the document field name as the first parameter and the class
+you want to cast the results to as the second parameter as shown below:
+
+.. code-block:: java
+
+   collection.distinct("countries", String.class);
+
+You can specify a field on the document or one within an **embedded document**
+using **dot notation**. The following method call returns each distinct
+value of the ``wins`` field in the ``awards`` embedded document:
+
+.. code-block:: java
+
+   collection.distinct("awards.wins", Integer.class);
+
+You can optionally pass a query filter to the method document to limit
+the set of documents from which the distinct values are retrieved as
+follows:
+
+.. code-block:: java
+
+   collection.distinct("type", Filters.eq("languages", "French"), String.class);
+
+The ``distinct()`` method returns an object that implements the
+``DistinctIterable`` interface. This interface contains methods to access,
+organize, and traverse the results. It also inherits methods from its parent
+interface, ``MongoIterable``, such as ``first()`` which returns the first
+result and ``cursor()`` which returns an instance of a ``MongoCursor``.
+
+
+Example
+-------
+
+The following snippet retrieves a list of distinct values for the ``year``
+document field from the ``movies`` collection. It uses a query document to
+match movies that include "Carl Franklin" as one of the values in the
+``directors`` array.
+
+.. include:: /includes/connect-guide-note.rst
+
+.. literalinclude:: /includes/usage-examples/code-snippets/Distinct.java
+  :language: java
```

Comment:
Looks like there's a missing space, though it doesn't seem to make a difference:
```suggestion
   :language: java
```